src/stylesheets/global.css

```diff
@@ -1,13 +1,71 @@
 /* Global settings and variables */
 
 @import '~normalize.css';
-@import url('https://fonts.googleapis.com/css?family=Lato:300i,300,400,700');
+@import url('https://fonts.googleapis.com/css2?family=Merriweather:wght@300&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Lato:wght@400;700&display=swap');
 
 :root {
   /* Colors */
-  --color-brand: #0025e4;
+  /* Interface Colors */
   --color-white: #fff;
+  --color-gray-300: #f9f9f9;
+  --color-gray-400: #e0e0e0;
+  --color-gray-500: #9e9e9e;
+  --color-gray-600: #888;
+  --color-gray-700: #666;
+  --color-gray-800: #484848;
+  --color-gray-900: #212121;
+  --color-black: #202020;
 
-  /* Fonts */
-  --font-family-base: 'Lato', 'Arial', sans-serif;
+  /* Message Colors */
+  --color-alert: #eb0000;
+  --color-warn: #ffbc00;
+  --color-success: #2bcf59;
+
+  /* Brand Colors */
+  --color-askdarcel-blue: #276ce5;
+  --color-sheltertech-blue: #0025e4;
+  --color-deep-blue: #202953;
+  --color-midnight-blue: #0f1428;
+
+  /* Font Families */
+  --font-family-hero: 'Merriweather', serif;
+  --font-family-headline: 'Merriweather', serif;
+  --font-family-title: 'Merriweather', serif;
+  --font-family-subtitle: 'Lato', sans-serif;
+  --font-family-caption: 'Lato', sans-serif;
+  --font-family-action: 'Lato', sans-serif;
+  --font-family-body: 'Lato', sans-serif;
+
+  /* Font Weights */
+  --font-weight-hero: 300;
+  --font-weight-headline: 300;
+  --font-weight-title: 300;
+  --font-weight-subtitle: 700;
+  --font-weight-caption: 700;
+  --font-weight-action: 800;
+  --font-weight-body: 400;
+
+  /* Font Sizes */
+  /* Web Type Scale */
+  --font-size-hero-w: 67px;
+  --font-size-headline-w: 50px;
+  --font-size-title-w: 30px;
+  --font-size-subtitle-w: 24px;
+  --font-size-caption-w: 18px;
+  --font-size-action-w: 14px;
+  --font-size-body-large-w: 30px;
+  --font-size-body-medium-w: 20px;
+  --font-size-body-small-w: 16px;
+
+  /* Mobile Type Scale */
+  --font-size-hero-m: 40px;
```

Comment:
We could actually do something pretty cool here by making use of media queries for the mobile font sizes, which means that these variables will have the correct values just based on the screen size. See https://css-tricks.com/responsive-designs-and-css-custom-properties-defining-variables-and-breakpoints/#using-css-variables-for-property-values for an article describing this, but it would look like the following:

```css

:root {
  --font-size-hero: 67px;
}

@media screen and (max-width: 600px) {
  :root {
    --font-size-hero: 40px;
  }
}